### ISSUE TITLE
Version conflict when installing via PIP

### ISSUE THREAD

**Issue by shogunbr** [collective/collective.jsonmigrator#49]
collective.jsonmigrator 2.0.0 is not installable via pip due ==2.7 constraint in python_requires
https://github.com/collective/collective.jsonmigrator/blob/78f7e92f9d2ad726e33a3eccd53f899ed31bbb4e/setup.py#L56

**Comment by wesleybl** [comment]
Yes this is a bug. Should be:

```
python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*!=3.5.*",
```

**Comment by wesleybl** [comment]
Fix in #50
